subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -59,8 +59,22 @@ For more information, see <<tar_tree_no_backing_file>>.
 The legacy `ArtifactTransform` API has been removed.
 For more information, see <<old_artifact_transforms_api>>.
 
+==== Legacy IncrementalTaskInputs API
+
+The legacy `IncrementalTaskInputs` API has been removed. For more information, see <<incremental_task_inputs_deprecation>>. This change also affects Kotlin Gradle Plugin and Android Gradle Plugin. With Gradle 8.0 you should use Kotlin Gradle Plugin 1.6.10 or later and Android Gradle Plugin 7.3.0 with `android.experimental.legacyTransform.forceNonIncremental=true` property or later.
+
 === Potential breaking changes
 
+==== Minimal supported Kotlin Gradle Plugin version changed
+With Gradle 7.x Kotlin Gradle Plugin 1.3.72 through 1.6.21 were tested and supported.
+With Gradle 8.x minimal Kotlin Gradle Plugin tested and supported version is 1.6.10.
+It is still possible to compile to lower Kotlin language version, by modifying language version and api version setting for the Kotlin compile task.
+
+==== Minimal supported Android Gradle Plugin version changed
+With Gradle 7.x Android Gradle Plugin 4.1 through 7.3 were tested and supported.
+With Gradle 8.x minimal Android Gradle Plugin tested and supported version is 7.3 with `android.experimental.legacyTransform.forceNonIncremental=true` property.
+Android Gradle Plugin 8 and later don't require additional property to work with Gradle 8.
```

Comment:
```suggestion
Gradle 7.x supports Android Gradle Plugin (AGP) 4.1 and above. AGP versions above 7.3 are not tested with Gradle 7.x.
Gradle 8.x supports AGP 8 and above.
Gradle 8.x supports AGP 7.3 and above if you configure the following property:
`android.experimental.legacyTransform.forceNonIncremental=true`
```

Suggestion: Active voice and parallel wording, as above. Reorder version support to mention the default supported version first, and the strings-attached support second.